CHANGELOG.md

```diff
@@ -23,6 +23,8 @@
   * Added `Transformer.cancel` method, which is called when the readable side of a `TransformStream` is cancelled or when its writable side is aborted.
   * Added `min` option to `ReadableStreamBYOBReader.read(view, options)`.
   * Added support for `AbortSignal.reason` when aborting a pipe.
+* 🚀 Buffers passed to `ReadableStreamBYOBReader.read(view)` will now be correctly [transferred](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer#transferring_arraybuffers)
+  if either `ArrayBuffer.prototype.transfer()` or `structuredClone()` is available. ([#136](https://github.com/MattiasBuelens/web-streams-polyfill/pull/135))
```

Comment:
:wave: Hi
There's a typo here, should have been pull/136.

Reply:
Woops! Thanks, should be fixed in https://github.com/MattiasBuelens/web-streams-polyfill/commit/8adb56689e440c858e693262ff200bf4ca390d7a. 🙂